### ISSUE TITLE
MS Word: нет возможности переключиться на раскладку

### ISSUE THREAD

**Issue by drakulavich** [tonsky/Universal-Layout#3]
При активном окне MS Word нельзя выбрать альтернативные раскладки (русскую и английскую)

**Comment by drakulavich** [comment]
Не помогло. Такая проблема только с несколькими приложениями пока: телеграм и ворд. Для остальных переключалка работает

**Comment by tonsky** [comment]
а что значит «нельзя выбрать?» Ворда нет, телеграм вроде работает

**Comment by drakulavich** [comment]
[Kapture 2018-10-29 at 19.36.50.webm.zip](https://github.com/tonsky/Universal-Layout/files/2525950/Kapture.2018-10-29.at.19.36.50.webm.zip)

Пардон за скомканное объяснение: нельзя переключить раскладку ни через шорткат, ни через меню выбора раскладки. Прикрепил скринкаст

**Comment by mkfallen** [comment]
Тоже есть такое, перезагрузка не помогает. У меня раскладки работают только в системных настройках и иногда в Файндере.
Причём если оставить системную английскую раскладку, то в других приложениях не получается включить ни английскую, ни русскую Универсал, ни шорткатом, ни через меню.

**Comment by mkfallen** [comment]
Оказалось, что указанная проблема наблюдается только в Universal Dark.bundle. А обычный выглядит нормально и на тёмной теме.

**Comment by tonsky** [comment]
Так, а в папке только один? Их вместе нельзя складывать :) Надо переделать

**Comment by mkfallen** [comment]
> Так, а в папке только один? Их вместе нельзя складывать :) Надо переделать
Да, сначала пробовал один Dark закинуть

**Comment by drakulavich** [comment]
Победил черную магию. Удалил бандл, скопировал заново. Поиграл с выбором раскладок при добавлении:
<img width="597" alt="russian-layout" src="https://user-images.githubusercontent.com/129260/47770284-312f8c80-dcf8-11e8-9508-f7771678b6f8.png">

Обе раскладки по непонятной причине дублируются в списке.

**Comment by h4** [comment]
Была аналогичная проблема с Телеграмом и Universal Dark. Обновился на v1.1 – проблема исчезла.

**Comment by drakulavich** [comment]
Обновился до 1.1. Дубликаты раскладок пропали. Закрываю тикет

**Comment by alekseysotnikov** [comment]
v1.3 Дубликаты раскладок присутствуют. В safari раскладки не переключаются совсем. 
<img width="519" alt="screenshot 2018-12-05 at 04 19 39" src="https://user-images.githubusercontent.com/1889355/49473574-ff549d00-f844-11e8-9b3e-42c46b446c2d.png">

**Comment by tonsky** [comment]
перелогинься?

**Comment by alekseysotnikov** [comment]
И перелогинивался и перезагружался. Непомогает. Mojave 10.14.1

**Comment by tonsky** [comment]
Ну я хз. Разберешься — расскажи :) Может у тебя и в /Library/Keyboard Layouts, и в ~/Library/Keyboard Layouts/ по копии?

**Comment by alekseysotnikov** [comment]
Вообщем пока ясно что, если копировать в /Library/Keyboard Layouts тоесть делать доступным эту раскладку для всех пользователей, полей с паролем и login окон, то дубликатов нет. 
А если копировать в ~/Library/Keyboard Layouts/ то появляются дубликаты.

А у тебя в Safari или Mail раскладка работает?

**Comment by alekseysotnikov** [comment]
Заработало после того как расковырял бандл и скопировал напрямую:

```
$ git clone https://github.com/tonsky/Universal-Layout.git
$ cd Universal-Layout/Universal.bundle/Contents/Resources/
$ sudo cp *Universal.* /Library/Keyboard\ Layouts/
```

**Comment by alex-mos** [comment]
Сейчас проверял — в macOS 10.14.1 во многих приложениях (things 3, bear) не переключается язык.
Обновил до 10.14.2 — всё нормально с переключением.

**Comment by alekseysotnikov** [comment]
Спасибо. Тоже обновился до 10.14.2, попробовал снова бандл, стало все ок с переключением и нет дубликатов.

**Comment by shvgn** [comment]
Получилось побороть непереключение в телеграме и сафари, если копировать в общий каталог. Когда было в домашнем, не работало. 
Макос 10.14.3, пробовал на свежей системе.

**Comment by Dota11222** [comment]
У знакомой не работает русская раскладка в телеграм на win 10, в системе стоит рус, но можно использовать только анг. Скачала последнюю версию с офф сайта. По данной теме сложно разобратся, так как далёк от программирования. Кто может обьяснить технический язык здесь: где находится /Library/Keyboard Layouts/ ? что такое Universal Dark.bundle? я гуглил, но не нашёл инфы и на анг пробовал. У кого есть какие советы?

**Comment by tonsky** [comment]
@Dota11222 это версия под Мак, на Винде работать не будет. Посмотри в README, там в конце есть альтернативы под Винду
